scripts/tests-ui-integration.sh

```diff
@@ -3,49 +3,26 @@
 PUBLISHED_FUEL_PACKAGE_NAME="${PUBLISHED_FUEL_PACKAGE_NAME-"fuels"}"
 PUBLISHED_NPM_TAG="${PUBLISHED_NPM_TAG-"next"}"
 
-# Versions
-FUEL_CORE_VERSION=$(cat ./internal/fuel-core/VERSION)
-FORC_VERSION=$(cat ./internal/forc/VERSION)
-TOOLCHAIN="CI"
-
 # Project
-ROOT_DIR=$(pwd)
-PLAYWRIGHT_DIR="$ROOT_DIR"
-PROJECT_DIR="$ROOT_DIR/test-project"
-
-echo "1. Install toolchains"
-if [ -x "$(command -v fuelup)" ]; then
-  echo "Fuelup exists"
-else
-  echo "Fuelup does not exist - installing 'fuelup'"
-  curl -fsSL https://install.fuel.network/ | sh -s -- --no-modify-path
-  export PATH="${HOME}/.fuelup/bin:${PATH}"
-fi
+REPO_DIR=$(pwd)
+PATH="${REPO_DIR}/internal/fuel-core/fuel-core-binaries:${PATH}"
+PATH="${REPO_DIR}/internal/forc/forc-binaries:${PATH}"
 
-fuelup toolchain new $TOOLCHAIN
-fuelup default $TOOLCHAIN
-fuelup component add fuel-core@$FUEL_CORE_VERSION
-fuelup component add forc@$FORC_VERSION
+PROJECT_DIR="$(mktemp -d)/integration-test-project"
+echo "$PROJECT_DIR"
 
-echo "2. Scaffold a new project with '$PUBLISHED_FUEL_PACKAGE_NAME@$PUBLISHED_NPM_TAG'"
+echo "1. Scaffold a new project with '$PUBLISHED_FUEL_PACKAGE_NAME@$PUBLISHED_NPM_TAG'"
 if [ -d "$PROJECT_DIR" ]; then
   echo "Removing existing project directory '$PROJECT_DIR'"
-  rm -rf $PROJECT_DIR
+  rm -rf "$PROJECT_DIR"
 fi
-pnpm create $PUBLISHED_FUEL_PACKAGE_NAME@$PUBLISHED_NPM_TAG $PROJECT_DIR --no-install
-
-echo "3. Intialise the project"
-cd $PROJECT_DIR
-pnpm remove fuels
-pnpm add $PUBLISHED_FUEL_PACKAGE_NAME@$PUBLISHED_NPM_TAG
-pnpm  --ignore-workspace install
-cp .env.example .env.local
+NPM_CONFIG_REGISTRY="https://npm-packages.fuel.network" VITEST="to-not-install-fuelup" pnpm create "$PUBLISHED_FUEL_PACKAGE_NAME"@"$PUBLISHED_NPM_TAG" "$PROJECT_DIR"
 
-echo "4. Running UI tests"
-cd $ROOT_DIR
+echo "2. Running UI tests"
+cd "$REPO_DIR"
 PROJECT_DIR=$PROJECT_DIR sh ./scripts/tests-ui.sh
 TEST_RESULT=$?
 
-rm -rf $PROJECT_DIR
+rm -rf "$PROJECT_DIR"
```

Comment:
1. What are the double quotes fixing?
1. Also, please review/add comments to this file.

Reply:
1. I'm using [shellcheck](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck) in my editor and, as I was rewriting the code, it guided me to wrap it in double quotes because of [SC2086](https://www.shellcheck.net/wiki/SC2086) so I did it instinctively and planned to revert it later as it doesn't have any bearing in this scenario because the `PROJECT_DIR` path doesn't have any spaces in it. We might want to integrate shellcheck for linting our scripts, though.
2. Added comments in https://github.com/FuelLabs/fuels-ts/pull/3749/commits/715959e5fdcfa55047ee9b86fc331a46cf6967d8. Will write comments on the removed code why I removed it.

Reply:
Ah I see, no worries.

I don't believe the double quotes would cause any trouble, in which case we could keep them.